src/www/app/outline_server.ts

```diff
@@ -57,6 +60,11 @@ export class OutlineServer implements PersistentServer {
     return this.config.host;
   }
 
+  get usesUnsupportedCipher() {
+    return this.config.method === undefined ||
```

Comment:
Let's put this logic in a standalone function that can be reused.

Reply:
Done